tests/CoreFX/TopN.CoreFX.Windows.issues.json

```diff
@@ -0,0 +1,3686 @@
+                },
+                {
+                    "name": "System.Linq.Parallel.Tests.GroupJoinTests.GroupJoin_CustomComparator",
+                    "reason": "Xunit.Sdk.AllException Assert.All() Failure: 1 out of 1 items in the collection did not pass.\\r\\n[0]: Item: [0, System.Linq.Parallel.ListChunk`1[System.Int32]]\\r\\n     Xunit.Sdk.AllException: Assert.All() Failure: 2 out of 4 items in the collection did not pass.\\r\\n     [2]: Item: 0\\r\\n          Xunit.Sdk.EqualException: Assert.Equal() Failure\\r\\n          Expected: 8\\r\\n          Actual:   0\\r\\n             at Xunit.Assert.Equal[T](T expected, T actual, IEqualityComparer`1 comparer) in C:\\\\projects\\\\xunit\\\\src\\\\xunit.assert\\\\Asserts\\\\EqualityAsserts.cs:line 40\\r\\n             at Xunit.Assert.All[T](IEnumerable`1 collection, Action`1 action) in C:\\\\projects\\\\xunit\\\\src\\\\xunit.assert\\\\Asserts\\\\CollectionAsserts.cs:line 36\\r\\n     [0]: Item: 8\\r\\n          Xunit.Sdk.EqualException: Assert.Equal() Failure\\r\\n          Expected: 0\\r\\n          Actual:   8\\r\\n             at Xunit.Assert.Equal[T](T expected, T actual, IEqualityComparer`1 comparer) in C:\\\\projects\\\\xunit\\\\src\\\\xunit.assert\\\\Asserts\\\\EqualityAsserts.cs:line 40\\r\\n             at Xunit.Assert.All[T](IEnumerable`1 collection, Action`1 action) in C:\\\\projects\\\\xunit\\\\src\\\\xunit.assert\\\\Asserts\\\\CollectionAsserts.cs:line 36\\r\\n        at Xunit.Assert.All[T](IEnumerable`1 collection, Action`1 action) in C:\\\\projects\\\\xunit\\\\src\\\\xunit.assert\\\\Asserts\\\\CollectionAsserts.cs:line 45\\r\\n        at System.Linq.Parallel.Tests.GroupJoinTests.<>c__DisplayClass21_0.<GroupJoin_CustomComparator>b__3(KeyValuePair`2 p) in C:\\\\Users\\\\anandono\\\\source\\\\repos\\\\corefx\\\\src\\\\System.Linq.Parallel\\\\tests\\\\QueryOperators\\\\GroupJoinTests.cs:line 335\\r\\n        at Xunit.Assert.All[T](IEnumerable`1 collection, Action`1 action) in C:\\\\projects\\\\xunit\\\\src\\\\xunit.assert\\\\Asserts\\\\CollectionAsserts.cs:line 36"
+                },
+                {
+                    "name": "System.Linq.Parallel.Tests.GroupJoinTests.GroupJoin_Multiple",
+                    "reason": "Xunit.Sdk.AllExcep
```

Comment:
These tests were added in https://github.com/dotnet/corefx/pull/27930. They should be passing with current corefx. Are we running new tests on old corefx? We should be always picking up the corefx that is specified in https://github.com/dotnet/coreclr/blob/master/dependencies.props#L34

Reply:
The version dependencies are specified here - 

https://github.com/A-And/coreclr/blob/2d37073500eda78a10a322fadda31a38dc44aede/tests/src/Common/CoreFX/CoreFX.depproj

all of them pulled with a specific version of $(MicrosoftPrivateCoreFxNETCoreAppPackageVersion).
Those that are not specified in the .depproj come directly from the assemblies output in Core_Root (used to run CoreCLR tests). So I'm not entirely certain why these are failing.